File: app/src/main/java/com/tanka/accessories/todoroom/data/model/Note.java
package com.tanka.accessories.todoroom.data.model;

import androidx.room.ColumnInfo;
import androidx.room.Entity;
import androidx.room.PrimaryKey;

import java.io.Serializable;

/**
 * Created by access-tanka on 11/16/17.
 */

@Entity
public class Note implements Serializable {

    @PrimaryKey(autoGenerate = true)
    @ColumnInfo(name = "id")
    public int id;

    @ColumnInfo(name = "title")
    public String title;

    @ColumnInfo(name = "date")
    public String date;

    @ColumnInfo(name = "body")
    public String body;

    @ColumnInfo(name = "type")
    public String type;

    public int getId() {
        return id;
    }

    public void setId(int id) {
        this.id = id;
    }

    public String getTitle() {
        return title;
    }

    public void setTitle(String title) {
        this.title = title;
    }

    public String getDate() {
        return date;
    }

    public void setDate(String date) {
        this.date = date;
    }

    public String getBody() {
        return body;
    }

    public void setBody(String body) {
        this.body = body;
    }

    public String getType() {
        return type;
    }

    public void setType(String type) {
        this.type = type;
    }
}

File: app/src/main/java/com/tanka/accessories/todoroom/data/model/Note.kt
package com.tanka.accessories.todoroom.data.model

import androidx.room.ColumnInfo
import androidx.room.Entity
import androidx.room.PrimaryKey

@Entity
public data class Note(
        @PrimaryKey val id: Int,
        @ColumnInfo(name = "title") val title: String?,
        @ColumnInfo(name = "date") val date: String?,
        @ColumnInfo(name = "body") val body: String?,
        @ColumnInfo(name = "type") val type: String?
)
